src/js/components/Form/stories/ValidateOnBlur.js

```diff
@@ -43,6 +43,25 @@ export const ValidateOnBlur = () => (
           <TextInput name="email" aria-label="email" type="email" />
         </FormField>
 
+        <FormField
+          label="Size"
+          name="select-size"
+          htmlFor="select-size"
```

Comment:
```suggestion
          htmlFor="select-size__input"
```
small detail to properly associate label with input